src/app/shared/services/get-movies.service.ts

```diff
@@ -0,0 +1,143 @@
+import { FormingApiUrlService } from './auxiliaryServices/forming-api-url.service';
+import { SingleMovie } from '@shared/interfaces/singleMovie';
+import { Movies } from '@shared/interfaces/movies';
+import { Injectable } from '@angular/core';
+import { Observable, of, Subject, BehaviorSubject } from 'rxjs';
+import { catchError, tap, map } from 'rxjs/operators';
+import { HttpClient } from '@angular/common/http';
+import { MessageService } from './message.service';
+import { PostResponses } from '@shared/interfaces/postResponses';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+
+
+export class GetMoviesService {
+  constructor(
+    private http: HttpClient, 
+    private messageService: MessageService,
+    private formingApiUrlService: FormingApiUrlService
+    ) { }
+  
+  private searchInput: string = null;
+  searchTerms = new Subject<string>();
```

Comment:
Subjects are observables, so the should include $ at the end

Reply:
Done